generate.go

```diff
@@ -277,7 +276,7 @@ func (c *ConfigGenerator) GetRecords() (map[string]RecordA, map[string]RecordCna
 
 	// add metadata record
 	aRec := RecordA{
-		Answer: []string{rancherDNS},
+		Answer: strings.Split(*metadataAddress, ","),
```

Comment:
Probably need to trim here in case the input is comma+space separated like `1.2.3.4, 2.3.4.5`